src/Task1/searchForm.css

```diff
@@ -0,0 +1,29 @@
+.searchform_container {
+  display: "flex";
+  justify-content: "center";
+}
+
+.searchform_container > form {
+  width: 50%;
+  display: flex;
+  justify-content: space-around;
+}
+.searchform_container > form > input {
+  background: #323232;
+  outline: none;
+  border: none;
+  width: 70%;
+  height: 30px;
+  border-radius: 3px;
+  color: #ffffff;
+  padding-left: 10px;
+}
+.searchform_container > form > button {
+  background-color: #f65261;
+  height: 30px;
+  border-radius: 2px;
+  color: #ffffff;
+  outline: none;
+  border: none;
+  width: 25%;
+}
```

Comment:
When you use plain CSS with React, most of the traditional CSS guidelines still apply. In this case, you broke a common naming convention in CSS, which is to use `kebab-case`. You used `snake_case` here.

Note that this rule would be different for CSS modules, but that's another topic, and you are not using CSS modules (at least for now).

Reply:
yes, I fixed it also